camayoc/tests/qpc/api/v1/utils.py

```diff
@@ -10,8 +10,8 @@
     QPC_SCAN_TERMINAL_STATES,
 )
 from camayoc.exceptions import (
-    FailedScanException,
     WaitTimeError,
+    FailedScanException,
```

Comment:
Travis may complain about this.

Reply:
oops yup, I was working from a different computer than usual and had not installed that flake8 plugin